docs/man/docker-run.1.md

```diff
@@ -396,6 +396,21 @@ used in other containers using the **--volumes-from** option.
 read-only or read-write mode, respectively. By default, the volumes are mounted
 read-write. See examples.
 
+Labeling systems like SELinux require proper labels be placed on volume content
+mounted into a container, otherwise the secuirty system might prevent the
+processes running inside the container from using the content. By default,
+volumes are not relabeled.
+
+Two suffixes :z or :Z can be added to the volume mount. These suffixes tell
+Docker to relabel file objects on the shared volumes. The 'z' option tells
```

Comment:
Shared label looks something like
system_u:object_r:svirt_sandbox_file_t:s0

Private label looks like
system_u:object_r:svirt_sandbox_file_t:s0:c1,c2

Where c1,c2 is different for every container. Has a range of c1-c1024.